packages/gestalt/src/Video.js

```diff
@@ -477,6 +483,12 @@ export default class Video extends React.PureComponent<Props, State> {
             ))}
           <track kind="captions" src={captions} />
         </video>
+        {overlay && (
```

Comment:
So instead of all this custom HTML and stylings, I think im in favor of having either the `overlay` prop be of type `React.Node` so the user gets to choose what to show in this middle layer, or to utilize the `Video` component `children` for this purpose since that is not used right now.

I think long term this is a good option as it will open up the door for more overlaid UI from other teams such as post-complete video actions like follow buttons, next video thumbnails, or things like overlaid buttons for specific parts of a video clip.

Reply:
It's a great idea. PR updated as you suggested.